client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@automattic/react-i18n';
+import { useInterval } from 'lib/interval/use-interval';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+// Total time to perform "loading"
+const DURATION_IN_MS = 6000;
+
+/**
+ * This component is cloned from the CreateSite component of Gutenboarding flow
+ * to work with the onboarding signup flow.
+ */
+export default function ReskinnedProcessingScreen( { hasPaidDomain } ) {
```

Comment:
I think this UI component should be part of [@automattic/onboarding](https://github.com/Automattic/wp-calypso/tree/master/packages/onboarding) package and imported here, in Gutenboarding and in the new [editor launch flow](https://github.com/Automattic/wp-calypso/issues/43750) we're building.

It's not a blocker now but maybe it's worth leaving a note to import it from packages once the reusable component lands there.

Reply:
@razvanpapadopol As I commented above, we decided to keep the new processing screen under the signup components because we may need to remove the component when the variation group loses the testing.